docker-compose.yml

```diff
@@ -59,6 +59,9 @@ services:
       - 8080:8080
     depends_on:
       - rest-api
+    environment:
```

Comment:
**Note:** This will not work unless you all set up manifold and export the environment variables into a `.env` file in the project root.

@dbartalino @ej3076 @sloan-seamus 

@adriankalinowski I can invite you to our team on manifold if you'd like to have access to these variables.